src/qtel/CMakeLists.txt

```diff
@@ -70,6 +70,8 @@ install(TARGETS ${PROG} DESTINATION ${BIN_INSTALL_DIR})
 install(FILES connect.raw DESTINATION ${SHARE_INSTALL_PREFIX}/qtel/sounds)
 install(FILES qtel.desktop DESTINATION ${SHARE_INSTALL_PREFIX}/applications)
 install(FILES images/link.xpm DESTINATION ${SHARE_INSTALL_PREFIX}/icons)
```

Comment:
By the way, this is probably not the best location for xpm icons. The correct one should be `${SHARE_INSTALL_PREFIX}/pixmaps`. But I am not sure if this location is not hardcoded in the Qtel source code, therefore I have decided not to change it. Feel free to fix it if you want or let me know and I can fix it as a part of this PR.

Reply:
Done (as a second commit). :-)